cypress/e2e/views/migration-wave.view.ts

```diff
@@ -23,3 +23,5 @@ export const getSpecialMigrationWavesTableSelector = (
 ) => {
     return `table[aria-label="${columnSelector} table for migration wave ${wave.name}"]`;
 };
+
+export const waveTable = "table[aria-label='Migration waves table']";
```

Comment:
This should be inside the enum in order to have all selectors clearly grouped (line 14 of this same file)

Reply:
Is there an ID that we can use for the table ?

Reply:
No, there is no ID locator for this table at  the moment.